server/routes/items.js

```diff
@@ -38,14 +38,16 @@ router.post('/get', function(req, res){
                     return res.status(201).send({formattedItem, code: 201});
                 }
             });
+        } else {
+            return res.status(400).send({err: 'Type 2: Id is not a valid ObjectId', code: 403});
```

Comment:
This probably makes sense, but I might need an explanation, why are there two error types?